WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainTransferFooterView.swift

```diff
@@ -0,0 +1,104 @@
+
+        static let title = NSLocalizedString(
+            "register.domain.transfer.title",
+            value: "Looking to transfer a domain you already own?",
+            comment: "The title for the transfer footer view in Register Domain screen"
+        )
+        static let buttonTitle = NSLocalizedString(
+            "register.domain.transfer.button.title",
+            value: "Transfer domain",
+            comment: "The button title for the transfer footer view in Register Domain screen"
+        )
+    }
+
+    // MARK: - Views
+
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = WPStyleGuide.fontForTextStyle(.body, fontWeight: .regular)
+        label.textColor = UIColor.DS.Foreground.primary
+        label.numberOfLines = 2
+        label.adjustsFontForContentSizeCategory = true
+        return label
+    }()
+
+    private let primaryButton: UIButton = {
+        let button = FancyButton()
+        button.titleLabel?.font = WPStyleGuide.fontForTextStyle(.headline, fontWeight: .regular)
+        button.isPrimary = true
+        return button
+    }()
+
+    // MARK: - Init
+
+    init(configuration: Configuration) {
+        super.init(frame: .zero)
+        self.backgroundColor = UIColor(light: .systemBackground, dark: .secondarySystemBackground)
+        self.addTopBorder(withColor: .divider)
+        self.setup(with: configuration)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Setup
+
+    private func setup(with configuration: Configuration) {
+        let stackView = UIStackView(arrangedSubviews: [titleLabel, primaryButton])
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.spacing = Length.Padding.double
+        stackView.distribution = .fill
+
+        self.addSubview(stackView)
+
+        let insets = NSDirectionalEdgeInsets(
+            top: Length.Padding.double,
+            leading: Length.Padding.double,
+            bottom: Length.Padding.double,
+            trailing: Length.Padding.double
+        )
+
+        NSLayoutConstraint.activate([
+            stackView.topAnchor.constraint(equalTo: layoutMarginsGuide.topAnchor, constant: insets.top),
+            stackView.leadingAnchor.constraint(equalTo: readableContentGuide.leadingAnchor, constant: insets.leading),
+            readableContentGuide.trailingAnchor.constraint(equalTo: stackView.trailingAnchor, constant: insets.trailing),
+            safeAreaLayoutGuide.bottomAnchor.constraint(equalTo: stackView.bottomAnchor, constant: insets.bottom)
+        ])
+
+        let action = UIAction { _ in
+            configuration.buttonAction()
+        }
+        self.titleLabel.text = configuration.title
+        self.primaryButton.setTitle(configuration.buttonTitle, for: .normal)
+        self.primaryButton.addAction(action, for: .touchUpInside)
+    }
```

Comment:
This was originally written in SwiftUI. I rewrote it to UIKit to fix this [issue](https://github.com/wordpress-mobile/WordPress-iOS/pull/22016#pullrequestreview-1724742822). Technically it should be possible to fix that issue in SwiftUI by using `ignoresSafeArea(.keyboard)` modifier, but that doesn't work when embedding SwiftUI in UIKit. 🤷

Reply:
I'm surprised there's no solution to it